testing/features/support/components/navigation/default.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Navigation
+  class Default < ::Base
+    set_url "/iframe.html?id=components-navigation--default-story&viewMode=story"
+
+    element :navigation, ".js-cads-greedy-nav"
```

Comment:
I've tried to go for a more user based approach (eg find buttons by text), which means this class is basically empty.  Is that...ok?

Reply:
So if you need to directly use text. I'm not 100% for it, but if enough people decide to then I guess I'll have to cede.

2 things to note.
1) Do these text options change by language? If so should we be doing it?
2) We can still create POM elements using capybara locator args.

Essentially the declaration from item 3 onwards (The hashable args). Is a capybara locator strategy
`element` is the keyword method definition
`:symbol`, is the reference name
`PascalCaseClass` is the class (Only used in Section/Sections)
`{hash: args}` are then the arguments to locate. So here you could do something like `"button", text: "Hey there!"`

Reply:
Agreed to move to using the SitePrism elements